VideoWeb/VideoWeb/Controllers/ConsultationsController.cs

```diff
@@ -222,6 +226,23 @@ public async Task<IActionResult> StartConsultationAsync(StartPrivateConsultation
                     {
                         return Forbid();
                     }
+
+                    var johConsultationRoomLockedStatusKeyName = $"johConsultationRoomLockedStatus_{conference.Id}";
+                    var isLocked =
+                        await _distributedJohConsultationRoomLockCache.IsJOHRoomLocked(johConsultationRoomLockedStatusKeyName);
+
+                    if (isLocked)
+                    {
+                        Thread.Sleep(30000); //this is really not ideal, but I cant think of any other options, this should be done from kinly
```

Comment:
This is too long to pause the thread in my opinion, 30 seconds to keep the user hanging is not a good user experiece. If the redis cache lock is expires after 5 seconds (see above comment) then it should be no more than that long

Reply:
wait _consultationService.GetAvailableConsultationRoomAsync(request.ConferenceId, request.RoomType.MapToDomainEnum()).  Not an video-api n endpoint . I will reduce the thread delay however whilst demoing to komal i did not noticed any significant delay creation of room .  I will leave a 3 seconds delay .